include/struct_decls/struct_0206942C_decl.h

```diff
@@ -1,6 +1,6 @@
 #ifndef POKEPLATINUM_STRUCT_0206942C_DECL_H
 #define POKEPLATINUM_STRUCT_0206942C_DECL_H
 
-typedef struct UnkStruct_0206942C_t UnkStruct_0206942C;
+typedef struct RadarChain_t RadarChain;
```

Comment:
Repo style is to omit `_t` suffixes on struct types wherever possible; it's safe to declare this as `typedef struct RadarChain RadarChain;`

Additionally, is it possible to merge this and its definition into the `pokeradar.h` header? We're using transparent structs. Alternatively, merge the `typedef` and the definition into the same header within `include/struct_defs`.